### DISCUSSION TITLE
JCA generation with ant-javacard build chain

### DISCUSSION THREAD

**Discussion by jsteffl** [martinpaljak/ant-javacard#83] [Q&A] [answered]
Hi Martin and community,

Really awesome and powerful open source JavaCard build chain! May I ask if anybody ever succeeded with JCA generation with it? I'm not super fluent with Ant and Java on desktop, so I cannot spot the problem in the source code or being able to debug it (yet), but it looks to me that whatever "jckit" and "targetsdk" combination I choose in my build.xml, it never passes "JCA" argument to the JC Convertor call or it fails on the JCA file copy back to output path specified by "jca" attribute of the build task...

Any hint or confirmation of the bug would be highly appreciated!

Thanks a lot
Jan

**Comment by martinpaljak** [comment, marked as answer]
specify `<cap ... jca="yourapplet.jca">` and you should get your .jca file. Please give the 'next' branch a spin if possible (git checkout --recursive + ant should give ant-javacard.jar)

**Reply by jsteffl** [reply]
And that worked like a charm indeed! I was on old "latest" pre-build distro... Thanks a lot!
